### ISSUE TITLE
NTR: SCOT activity low in fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7935]
New term request SCOT activity low in fibroblasts (segal_050622154457): Succinyl-CoA acetoacetate transferase; Succinyl-CoA: 3-oxoacid-CoA transferase  UMLS: 5437007

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000361
- label: Reduced succinyl-CoA:3-oxoacid-CoA transferase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:11757586
- def: Activity of succinyl-CoA:3-oxoacid-CoA transferase (SCOT, or OXCT1; EC 2.8.3.5) below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7935
- synonyms:

**Comment by MickeySegal** [comment]
You already assigned this to https://hpo.jax.org/app/browse/term/HP:4000162

There have been a number of recent NTRs that seem to be getting processed for a second time.
